spec/support/ansible_shared/automation_manager/configuration_script.rb

```diff
@@ -162,19 +153,25 @@
       )
     end
 
-    it ".update_in_provider_queue" do
-      EvmSpecHelper.local_miq_server
-      params[:manager_ref] = 10
-      task_id = described_class.update_in_provider_queue(manager.id, params)
-      expect(MiqTask.find(task_id)).to have_attributes(:name => "Updating Ansible Tower Job Template")
-      expect(MiqQueue.first).to have_attributes(
-        :args        => [manager.id, params],
-        :class_name  => described_class.name,
-        :method_name => "update_in_provider",
-        :priority    => MiqQueue::HIGH_PRIORITY,
-        :role        => "ems_operations",
-        :zone        => manager.zone.name
-      )
+    context "Update through API" do
```

Comment:
I typically avoid adding a context when there is only one test inside it.  The `let`s defined here can be variables inside the `it`.